project/ui/qprof-ui.ipynb

```diff
@@ -93,8 +93,8 @@
     "    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">\n",
     "    </head>\n",
     "    <body>\n",
-    "    <a download href=\"{filepath}\" download>\n",
-    "    <button style=\"background-color: #007bff;\" class=\"lm-Widget jupyter-widgets jupyter-button widget-button\">Download File</button>\n",
+    "    <a download href=\"http://127.0.0.1:8889/lab/tree/ui/{filepath}\" download>\n",
```

Comment:
I kind of prefer:
`<a href=\"/lab/tree/ui/{filepath}\" download>\n"`
Can you try this?